packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagebubble/_index.scss

```diff
@@ -0,0 +1,31 @@
+.dx-chat-message-bubble-container {
+  display: flex;
+  flex-direction: column;
+  max-width: 70%;
+}
+
+.dx-chat-message-bubble {
+  width: max-content;
+}
+
+.dx-chat-message-group-alignment-start {
+  .dx-chat-message-bubble-container {
+    align-items: flex-start;
+  }
+
+  .dx-chat-message-bubble {
+    border-bottom-left-radius: 0;
```

Comment:
We don't support RTL mods right now. To properly use such variables we have to analyze the layouts, maybe we will disagree with the layout. I suggest we don't use these now, because there is a possibility that we will be fixing them later.